cmake/modules/FindOPENTRACE.cmake

```diff
@@ -0,0 +1,112 @@
+#.rst:
```

Comment:
could you rename the file to `FindOpenTracing.cmake` and rename the variables and target accordingly?